1.3.1 cabal install fail

```
cabal update; cabal install idris
```

failed with

```
[113 of 115] Compiling Idris.REPL       ( src/Idris/REPL.hs, dist/build/Idris/REPL.o )

src/Idris/REPL.hs:223:61: error:
    Variable not in scope:
      tupleToHostAddress
        :: (Integer, Integer, Integer, Integer) -> HostAddress
cabal: Leaving directory '/tmp/cabal-tmp-21014/idris-1.3.1'
cabal: Error: some packages failed to install:
idris-1.3.1 failed during the building phase. The exception was:
ExitFailure 1
```

Building from master got the same error.

GHC version: 8.0.2
OS: Gentoo Linux

--- a/idris.cabal
+++ b/idris.cabal
@@ -291,7 +291,7 @@ Library
                 , ieee754 >= 0.7 && < 0.9
                 , megaparsec >= 6.2 && < 7
                 , mtl >= 2.1 && < 2.3
-                , network < 2.8
+                , network >= 2.7 && < 2.9
                 , optparse-applicative >= 0.13 && < 0.15
                 , pretty < 1.2
                 , process < 1.7

